### DISCUSSION TITLE
Error trying to build and run the project.

### DISCUSSION THREAD

**Discussion by CarlossDMC** [dbeaver/dbeaver#34277] [DBeaver Development]
I followed all the steps in the wiki, but when I run the project, the screen shows up and then closes, and I encounter this problem.

![image](https://github.com/dbeaver/dbeaver/assets/83795819/116f4e18-c947-4ee4-aac3-aaad67bfd240)

![image](https://github.com/dbeaver/dbeaver/assets/83795819/a0956ae7-438e-447d-9d7b-383111cb1346)

![image](https://github.com/dbeaver/dbeaver/assets/83795819/04a18f7f-2a36-4e6f-9543-e585ebe57187)

![javaw_ZTdsN3khxy](https://github.com/dbeaver/dbeaver/assets/83795819/df4fe188-976b-4431-9eab-d7b2830d052e)

[.log](https://github.com/user-attachments/files/15598622/default.log)

I already built the project many times, but the error persists ):

**Comment by TB-HunBoka** [comment]
I tried to do the same for 3 days now, but I was unable to import the run configuration.
But looks like your eclips missing some plugins.
Can you check if the required version of jacoco is installed in the plugins tab?
![image](https://github.com/dbeaver/dbeaver/assets/126617861/609f9111-9caf-43b8-a9fe-2f06a0d387c1)

**Reply by TB-HunBoka** [reply]
Finaly I was able to import the run configuration correctly.
I had a different problem, but I had to check the "Include required Plig-ins automatically while launching" option in the run configuration's plug-ins tab.
Give it a try.

**Reply by CarlossDMC** [reply]
Hi, the tip that you gave me helped to solve that problem, but I'm still having some problems with dependencies. However, I'm going to continue trying to build. I really want to improve the product with my code.

Thanks a lot.🫡

**Reply by serjiokov** [reply]
Hi, the simplest way to switch Launch Configuration to feature set than specify main product feature next click on add required button in order automatically add dependent features, next use validate button to check dependency resolved. Other plugins can be added through your feature or just use advance plugins button.
